src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/CommonModels/Utils.cs

```diff
@@ -163,6 +163,10 @@ public static ContainerType GetPsContainerType(string containerType)
             {
                 return ContainerType.Windows;
             }
+            else if (containerType == ContainerType.AzureSqlContainer.ToString())
```

Comment:
Shouldn't this comparison be done against the corresponding ContainerType of the Hydra DLL?